Controllers/controllers.js

```diff
@@ -79,4 +79,16 @@ exports.postCommentByArticle = (req,res,next) => {
     .catch(err => {
         next(err)
     })
+}
+
+
+exports.deleteCommentById = (req,res,next) => {
+    const id = req.params.comment_id
+    removeCommentById(id)
+    .then(() => {
+        res.status(204).send()
```

Comment:
if there is nothing to send back you can use the .end() instead of .send(). It will just end the connection